packages/dashboard/src/components/DashboardGridLayout/AddCardPanel/AddCard/AddCard.types.ts

```diff
@@ -0,0 +1,61 @@
+import { IStyle } from 'office-ui-fabric-react/lib/Styling';
+
+export interface IAddCardProps {
+  id: string;
```

Comment:
>id: string; [](start = 2, length = 11)

write a regular comment informing this is auto generated in the code and what users pass will not be honored